src/sidebar/app.js

```diff
@@ -37,6 +38,10 @@ if (settings.raven) {
   angular.module('ngRaven', []);
 }
 
+if(settings.googleAnalytics){
```

Comment:
Nit: There should be spaces around the condition of an if statement, ie. `if (cond) {`. I thought this was included in the ESLint config, apparently that needs adding.